setup.py

```diff
@@ -142,7 +142,7 @@ def run_tests(self):
     platforms=["any"],
     classifiers=CLASSIFIERS,
     python_requires=">=3.6",
-    install_requires=["pymongo>=3.4,<=4.0"],
+    install_requires=["pymongo>=3.4,<=5.0"],
```

Comment:
Change to <5.0 , so the day 5 gets out, we can make sure it is supported before accepting it

Reply:
good catch, didn't look close enough at what was there already